### ISSUE TITLE
Please update dependency astw to 1.3.0

### ISSUE THREAD

**Issue by jimlloyd** [substack/lexical-scope#20]
Per the issue I filed in astw that you have already resolved: https://github.com/substack/astw/issues/6
It looks like lexical-scope needs to be bumped to depend on astw@1.3.0.

**Comment by jimlloyd** [comment]
And if I am reading my npm-debug.log correctly, it looks like you may need to update insert-module-globals to reference the new version of lexical-scope, once it is published.

**Comment by jimlloyd** [comment]
My problem has somehow resolved itself. I had discovered that I could reproduce the problem by trying to `npm install cucumber` in an empty directory. I have TJ's module n installed to manage multiple installations of node. Earlier in the day I had already tried switching for node 0.12 to node 0.11.15 with no luck. But I then tried switching to node 0.10.35 and was able to install cucumber. I switched back to 0.12 and was again able to install. I went back to my main project, and could again do the install. Furthermore, I see that lexical-scope@1.1.0 and astw@1.1.0 are both installed, with the suspect version of esprima-fb:

```
$ npm list | grep --context=2 esprima
│ │ │ ├─┬ lexical-scope@1.1.0
│ │ │ │ └─┬ astw@1.1.0
│ │ │ │   └── esprima-fb@3001.1.0-dev-harmony-fb
│ │ │ ├── process@0.6.0
│ │ │ └── through@2.3.6
```

(other uses of esprima omitted in the above).

So, I think the problem I experienced earlier was a red herring. But in my googling to resolve this problem I have seen other cases where people had mysterious difficulty with esprima-fb, so thought I should document my experience here just in case.

Closing this issue.
Thanks for your earlier help with astw.
